javaparser-core-serialization/src/main/java/com/github/javaparser/serialization/JavaParserJsonDeserializer.java

```diff
@@ -42,13 +43,33 @@
  * Deserializes the JSON file that was built by {@link JavaParserJsonSerializer}.
  */
 public class JavaParserJsonDeserializer {
-
+    public static final String COMMENT_PROPERTY_KEY =
+            CommentMetaModel.NAME.substring(0, 1).toLowerCase() + CommentMetaModel.NAME.substring(1);
```

Comment:
Nice! There's a util for capitalization in https://github.com/javaparser/javaparser/blob/master/javaparser-core/src/main/java/com/github/javaparser/utils/CodeGenerationUtils.java